ArduCopter/autoyaw.cpp

```diff
@@ -56,6 +56,7 @@ void Mode::AutoYaw::set_mode(autopilot_yaw_mode yaw_mode)
     if (_mode == yaw_mode) {
         return;
     }
+    _last_mode = (autopilot_yaw_mode)_mode;
```

Comment:
should probably just make `_mode`  a `autopilot_yaw_mode` type

Reply:
Agreed. Done.  Added as seperate commit to keep it clear.